apps/gateway/src/common/routes.ts

```diff
@@ -4,6 +4,10 @@ const ROUTES = {
     Base: '/api',
     V1: {
       Base: '/v1',
+      DocumentLogs: {
+        Base: '/document-logs',
+        AddMessage: '/:documentLogUuid/add-message',
+      },
```

Comment:
We could expose this just as `/chat`, user does not need to know this is a "document log".

Reply:
`/chat/:documentLogUuid/add-message`? or `chat/add-message` and `documentLogUuid` as a body param?

Reply:
> `/chat/:documentLogUuid/add-message`?

yeah, just a change in the name so that it is more straightforward to the user. I think 'chat' is easier to understand than 'document-logs' to the user, since this is the only use they'll give it through the api.